build.sbt

```diff
@@ -578,10 +593,16 @@ def customCommands: Seq[Setting[_]] = Seq(
     "clean" ::
       "checkCredentials" ::
       "conscript-configs" ::
+      "interfaceProj/compile" :: // Java project needs to compile first
```

Comment:
This may or may not be needed with the latest sbt-doge. I needed to get a build out first then fix doge.